06.bowling_object/frame.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  LAST_FRAME = 9
+
+  attr_reader :shots
+
+  def initialize(frame_idx, marks)
+    @shots = []
+    @frame_idx = frame_idx
```

Comment:
```suggestion
    @frame_idx = frame_idx
    @shots = []
```

わずかな違いですがこっちの方が `@shots`の宣言と`<<`が近くなってベターです。
https://bootcamp.fjord.jp/pages/311